oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBTimeable.java

```diff
@@ -0,0 +1,40 @@
+package org.heigit.ohsome.oshdb;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+public interface OSHDBTimeable {
+  
+  long getEpochSecond();
+  
+  default OSHDBTimestamp getTimestamp() {
+    return new OSHDBTimestamp(getEpochSecond());
+  }
+  
+  default boolean isBefore(OSHDBTimeable other) {
+    return getEpochSecond() < other.getEpochSecond();
+  }
+  
+  static boolean isBefore(OSHDBTimeable a, OSHDBTimeable b) {
```

Comment:
are this static method and the corresponding `static boolean isAfter` really needed?